govuk-mobile-ios-onboarding/govuk-mobile-ios-onboarding/AppDelegate.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  AppDelegate.swift
+//  govuk-mobile-ios-onboarding
+//
+//  Created by Thomas Bates on 18/04/2024.
+//
```

Comment:
Seems somewhat redundant given the git history and context.

Reply:
Not sure what has happened here, this code is out of date. Same as the other repo, I will remove headers going forward